deploy/monitor/grafana_dashboard.yaml

```
@@ -0,0 +1,511 @@
+          },
+          {
+            "type": "panel",
+            "id": "singlestat",
+            "name": "Singlestat",
+            "version": ""
+          }
+        ],
+        "annotations": {
+          "list": [
+            {
+              "builtIn": 1,
+              "datasource": "-- Grafana --",
+              "enable": true,
+              "hide": true,
+              "iconColor": "rgba(0, 211, 255, 1)",
+              "name": "Annotations & Alerts",
+              "type": "dashboard"
+            }
+          ]
+        },
+        "description": "Application metrics",
+        "editable": true,
+        "gnetId": null,
+        "graphTooltip": 0,
+        "links": [],
+        "panels": [
+          {
+            "collapsed": false,
+            "gridPos": {
+              "h": 1,
+              "w": 24,
+              "x": 0,
+              "y": 0
+            },
+            "id": 9,
+            "panels": [],
+            "repeat": null,
+            "title": "Uptime",
+            "type": "row"
+          },
+          {
+            "cacheTimeout": null,
+            "colorBackground": false,
+            "colorValue": true,
+            "colors": [
+              "#d44a3a",
+              "rgba(237, 129, 40, 0.89)",
+              "#299c46"
+            ],
+            "datasource": "Prometheus",
+            "decimals": null,
+            "format": "percent",
+            "gauge": {
+              "maxValue": 100,
+              "minValue": 80,
+              "show": true,
+              "thresholdLabels": true,
+              "thresholdMarkers": true
+            },
+            "gridPos": {
+              "h": 8,
+              "w": 3,
+              "x": 0,
+              "y": 1
+            },
+            "id": 8,
+            "interval": null,
+            "links": [],
+            "mappingType": 1,
+            "mappingTypes": [
+              {
+                "name": "value to text",
+                "value": 1
+              },
+              {
+                "name": "range to text",
+                "value": 2
+              }
+            ],
+            "maxDataPoints": 100,
+            "nullPointMode": "connected",
+            "nullText": null,
+            "postfix": "",
+            "postfixFontSize": "50%",
+            "prefix": "",
+            "prefixFontSize": "50%",
+            "rangeMaps": [
+              {
+                "from": "null",
+                "text": "N/A",
+                "to": "null"
+              }
+            ],
+            "sparkline": {
+              "fillColor": "rgba(31, 118, 189, 0.18)",
+              "full": false,
+              "lineColor": "rgb(31, 120, 193)",
+              "show": false
+            },
+            "tableColumn": "",
+            "targets": [
+              {
+                "expr": "((sum(avg_over_time(up{job='unifiedpush-operatorr'}[1m])))/(count(avg_over_time(up{job='unifiedpush-operator'}[30m]))))*100",
```

typo here
```suggestion
                "expr": "((sum(avg_over_time(up{job='unifiedpush-operator'}[1m])))/(count(avg_over_time(up{job='unifiedpush-operator'}[30m]))))*100",
```

---

Done 👍

---

found that as the time varies the above command gave less consistence in results. 
I came to the conclusion that this is a better way of working out the average uptime
https://github.com/aerogear/mobile-security-service-operator/blob/bcd6150726322ea74341ca8cf1225f14611c66a6/deploy/monitor/mss_grafana_dashboard.yaml#L145

So suggest changes

```suggestion
                "expr": "(avg(up{job='unifiedpush-operator'}))*100",
```